modules/vnet/main.tf

```diff
@@ -0,0 +1,6 @@
+resource "azurerm_virtual_network" "vnet" {
+  name                = "dmzVnet"

```

Comment:
Since the default is dmzVnet, you can make this a variable and omit it if that's the name you want.